lib/rbs/prototype/rb.rb

```diff
@@ -15,6 +15,14 @@ def method_kind
             :instance
           end
         end
+
+        def attribute_kind
```

Comment:
https://github.com/ruby/rbs/blob/8871f6da72bd8c4c3e08dbd52dcc9bd7820e1512/lib/rbs/parser.y#L205-L207
Attribute doesn't support `singleton_instance` syntax (`self?`), unlike method definition. Is it intentional?

Reply:
It's intentional but I'm open to adding the syntax if it's better.

I was assuming that we rarely define attributes for both singleton and instance while the presence of `module_function` implies it happens frequently for`def`s.

Reply:
> I was assuming that we rarely define attributes for both singleton and instance while the presence of module_function implies it happens frequently for `def`s.


I completely agree. It is rare and I do not have any use cases for `singleton_instance` attributes.